content/en/integrations/guide/prometheus-metrics.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Mapping Prometheus Metrics to Datadog Metrics
+kind: guide
+aliases:
+  - /integrations/faq/how-to-collect-metrics-with-sql-stored-procedure/
+further_reading:
+- link: "https://www.datadoghq.com/blog/sql-server-metrics/#create-a-stored-procedure-to-generate-and-collect-metrics"
+  tag: "Blog"
+  text: "Create a stored procedure to generate and collect metrics"
+- link: "/integrations/mysql/"
+  tag: "Documentation"
+  text: "Datadog-MySQL integration"
+---
+
+If you are using Datadog's Prometheus or OpenMetrics checks, you may be interested in how these metrics map to existing Datadog metrics types. 
+
+## Prometheus and OpenMetrics metric types
```

Comment:
Should we also link to https://prometheus.io/docs/concepts/metric_types/#counter and others?